prov/gni/src/gnix_msg.c

```diff
@@ -2573,6 +2621,8 @@ ssize_t _gnix_recv(struct gnix_fid_ep *ep, uint64_t buf, size_t len,
 		req->msg.recv_md[0] = md;
 		req->msg.send_iov_cnt = req->msg.recv_iov_cnt = 1;
 		req->msg.recv_flags = flags;
+		if (flags & FI_MULTI_RECV)
+			req->msg.recv_flags |= GNIX_MSG_MULTI_RECV_SUP;
```

Comment:
I remember talking to Evan about this.  Does FI_MULTI_RECV even make sense with fi_recvv?  Do we need to document that this is not supported, or is this somewhere in the man pages?